src/test/java/io/crate/operation/scalar/ClassnamerFunctionTest.java

```diff
@@ -59,17 +59,24 @@
 
     private void registerPlugin(Settings settings) throws Exception {
         ScalarFunctionModule scalarFunctionModule = new ScalarFunctionModule();
+        TableFunctionModule tableFunctionModule = new TableFunctionModule();
```

Comment:
otherwise it throws 
```
1) No implementation for java.util.Map<java.lang.String, io.crate.metadata.tablefunctions.TableFunctionImplementation> was bound.
```